jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -724,7 +724,7 @@ private static final class Component<T extends PlatformManagedObject> {
 		private static final String OPENJ9_DIAGNOSTICS_MXBEAN_NAME = "openj9.lang.management:type=OpenJ9Diagnostics"; //$NON-NLS-1$
 
 		static void registerAll() {
-			// register standard singleton beans
+			/* Register standard singleton beans for JDK8. */
```

Comment:
I'm not thrilled that you changed many `//` style comments to `/* ... */`, but I won't block this change.

Reply:
Reverted.